src/modules/SX127x/SX127x.h

```diff
@@ -817,6 +818,7 @@ class SX127x: public PhysicalLayer {
       \brief Interrupt-driven receive method. DIO0 will be activated when full valid packet is received.
       \param len Expected length of packet to be received, or 0 when unused.
       Defaults to 0, non-zero required for LoRa spreading factor 6.
+      If non-zero for LoRa spreading factor > 6, RxSingle is used and value must be given in symbols.
```

Comment:
In all honesty, I don't really understand what you mean / how I should improve this. Is this something you can do, maybe?

Reply:
Reality is that 90% or more of LoRaWAN stack developers won't know or care about symbols. Some developers will find air time calcs useful but that's a tiny percentage who are pushing the envelope on duty cycle. The vast majority will have an uplink interval so benign that airtime/symbols/duty cycle/TTN FUP etc etc aren't an issue.

I'd move it to a protected function so that it is available if need be and so doesn't become a support item that can't be realistically answered in just a few sentences.

Reply:
I implemented this by slightly changing `SX127x::startReceive` so that it accepts an actual timeout as its first argument. Honestly the way that it was done previously (hijackign the argument to store the Rx mode) was rather convoluted. Now it more closely matches `SX126x::startReceive`.